### DISCUSSION TITLE
What should be the new pithy tagline for next version?

### DISCUSSION THREAD

**Discussion by 640-KB** [640-KB/GLaBIOS#14] [Polls]
**Thank you to everyone who voted on this round.  It was a difficult decision, so rather than having to choose only one GLaBIOS will now choose a different tagline on every reboot, based on mood.**

This poll will continue to accept new suggestions for the next version yet.

During development the POST screen felt a little empty so some placeholder text ("The hero we need but don't deserve") was added to fill it out until something better could be written.  As often happens, nothing better was ever really thought up, so it just sort of stuck.  It was suggested that this could change for each major version of GLaBIOS which sounds like a great idea.

So we're looking for suggestions for one for the next version.  Here are some of the proposed taglines, but are by no means a finalized list - so please add any other ideas below!

Some guidelines/suggestions:
- Must be 39 characters or less
- Should not have any actual meaning
- Can be distant or obscure references at historical PCs
- Ideally avoid direct references to "the game"
- The positioning should feel like a branding tagline (as in, it will be preceded by the word "GLaBIOS")

**Comment by mdhampton** [comment]
What, me worry?

**Reply by PickledDog** [reply]
A few to choose from.
Now asbestos-free
Soon to be a motion picture (or Now in Theaters)
Game of the Year Edition
This time, it's personal
Abort, Retry, Succeed
